pkg/internal/netolly/export/prom/expirer_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/grafana/beyla/pkg/internal/netolly/ebpf"
 )
 
-const timeout = 3 * time.Second
+const timeout = 30000 * time.Second
```

Comment:
Just checking if this was changed on purpose?

Reply:
No  😅 Reverting it